web-console/src/components/show-log.tsx

```diff
@@ -43,15 +43,17 @@ export interface ShowLogProps extends React.Props<any> {
 
 export interface ShowLogState {
   logValue: string;
+  tail: boolean;
 }
 
 export class ShowLog extends React.Component<ShowLogProps, ShowLogState> {
+  log = React.createRef<HTMLTextAreaElement>();
```

Comment:
could you denote with with a `public` prefix and also add a empty line after this one?

Reply:
changed

Reply:
<img width="537" alt="Screen Shot 2019-05-20 at 5 08 18 PM" src="https://user-images.githubusercontent.com/37322608/58059475-eb1a0900-7b21-11e9-9687-de20c9e87510.png">

Ts lint isn't onboard with that, should I do it anyway